src/__test__/components/RatingComponent.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import StarRatings from 'react-star-ratings';
+import RateArticle from '../../components/Rating/RatingComponent';
+import { rateArticleAction, getRatings } from '../../actions/rating.action';
+
+let wrapper = shallow(<RateArticle slug="hi-there" rating={4} averageRating={3} getRatings={getRatings} rateArticleAction={rateArticleAction} />);
```

Comment:
Have these props defined in a constant then destructure them when defining your component